client/src/pages/ManageBookings/ManageBookings.tsx

```diff
@@ -0,0 +1,98 @@
+import DateFnsUtils from '@date-io/date-fns';
+import { Box, Grid, Paper, Typography, useTheme } from '@material-ui/core';
+import { DatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+import { MaterialUiPickersDate } from '@material-ui/pickers/typings/date';
+import { cloneElement, useState } from 'react';
+import { CurrentBookings, NextBooking, PastBookings } from '../../components/ManageBookings/';
+import { checkIfBooked } from '../../helpers/ManageBookings';
+import mockBookingData from './mockBookingData';
+import { BookingRequests } from './types';
+import useStyles from './useStyles';
+
+export default function ManageBookings(): JSX.Element {
+  const classes = useStyles();
+  const theme = useTheme();
+  const [date, setDate] = useState<Date | null>(new Date());
+  const [sitterBookings, setSitterBookings] = useState<BookingRequests>(mockBookingData);
+
+  const updateDate = (date: MaterialUiPickersDate) => {
+    setDate(date);
+  };
+
+  const renderDay = (
+    day: MaterialUiPickersDate,
+    selectedDate: MaterialUiPickersDate,
+    dayInCurrentMonth: boolean,
+    dayComponent: JSX.Element,
+  ) => {
+    const renderDay = day?.getTime();
+    const selectedDay = selectedDate?.getTime();
+    const isSelectedDay = selectedDay === renderDay;
+
+    const isBookedDay = checkIfBooked(sitterBookings, day);
+
+    const dayStyle = {
+      style: { backgroundColor: theme.palette.common.white, border: 'none', color: theme.palette.common.black },
+      disabled: true,
+    };
+
+    if (isBookedDay) {
+      dayStyle.style.backgroundColor = theme.palette.primary.main;
+      dayStyle.disabled = false;
+    }
+    if (isSelectedDay) {
+      dayStyle.style.border = `2px solid ${theme.palette.grey[400]}`;
+    }
+    const dayComponentStyled = cloneElement(dayComponent, dayStyle);
+    return <div>{dayComponentStyled}</div>;
```

Comment:
Please use MUI `Box` component for this `div`

Reply:
resolved by decf3104dfffa92f05635b7a287a416bcf074dd3
Actually changed the div to a react fragment, because div / box was unnessary.